beta/do-app-platform-op-cli/README.md

```diff
@@ -0,0 +1,145 @@
+# [Beta] Deploy 1Password SCIM bridge on DigitalOcean App Platform with 1Password CLI
+
+This deployment example describes how to deploy 1Password SCIM bridge as an app on DigitalOcean's [App Platform](https://docs.digitalocean.com/products/app-platform/) service using [1Password CLI](https://developer.1password.com/docs/cli), the DigitalOcean command line interface ([`doctl`](https://docs.digitalocean.com/reference/doctl/)), and the DigitalOcean [1Password Shell Plugin](https://developer.1password.com/docs/cli/shell-plugins/).
+
+The app consists of two [resources](https://docs.digitalocean.com/glossary/resource/): a [service](https://docs.digitalocean.com/glossary/service/) for the SCIM bridge container and an [internal service](https://docs.digitalocean.com/glossary/service/#internal-services) for Redis.
+
+## In this folder
+
+- [`README.md`](./README.md): the document that you are reading. 👋😃
+- [`op-scim-bridge.yaml`](./op-scim-bridge.yaml): an App Platform [app spec](https://docs.digitalocean.com/glossary/app-spec/) for 1Password SCIM bridge [templated with secret references](https://developer.1password.com/docs/cli/secrets-template-syntax) to load secrets from your 1Password account.
+
+## Overview
+
+Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
+
+- For standard deployments, App Platform will host your SCIM bridge for a predictable cost of $10 USD/month (at the time of last review).
+- You do not need to manage a DNS record. DigitalOcean automatically provides a unique URL for your SCIM bridge.
```

Comment:
We should include a point about how DO handles the certificate management for this DNS record as a benefit as well.

Reply:
Noted in the latest commit. Thanks for the review!